test/issues/general/test_5703.test

```diff
@@ -0,0 +1,19 @@
+# name: test/issues/general/test_5703.test
+# description: Issue 5703: Can't have a window aggregate and unnest.
+# group: [general]
+
+
+statement error
+select unnest([5,4,3]) as r, r as x, sum(r) OVER ();
+----
+Binder Error
+
+
+statement error
+SELECT unlist(str_split('A',' ')) AS "r", CASE WHEN ("r" = 'A') THEN (-1) ELSE 0 END AS "x", sum("x") OVER () AS "s";
+----
+Binder Error
+
+# check database is not invalidated
```

Comment:
This is not necessary - an `InternalError` always fails a test (even if the test is labeled as `statement error`)